deviceplugin/usb.go

```diff
@@ -165,11 +166,13 @@ func queryUSBDeviceCharacteristicsByDirectory(fsys fs.FS, path string) (result *
 		return result, err
 	}
 
+	serial := ""
 	serBytes, err := fs.ReadFile(fsys, filepath.Join(path, usbDevicesDirSerialFile))
-	if err != nil {
+	if !errors.Is(err, fs.ErrNotExist) && err != nil {
 		return result, err
+	} else {
```

Comment:
This `else` layers the possibility that the error is not nil AND it's that the file doesn't exist. That's OK because then the `serBytes` is simply empty?

Reply:
In the case that the error is not nil and that the file doesn't exist `serBytes` would be nil. Added a check for this in the rebase